terasoluna-gfw-common-libraries/pom.xml

```diff
@@ -175,5 +175,17 @@
         <maven.javadoc.skip>true</maven.javadoc.skip>
       </properties>
     </profile>
+    <profile>
+      <id>jdk17</id>
+      <activation>
+        <jdk>17</jdk>
+      </activation>
+      <properties>
+        <!-- coverall 4.3.0 or previous version does not work on Java 11, see https://github.com/trautonen/coveralls-maven-plugin/issues/112 -->
```

Comment:
- [ ] The comments seem to be wrong here too.

Reply:
Comment has been corrected.
9fd678d583bcba63d319042a2943db9641a3d4e9